### DISCUSSION TITLE
Shwoing api response in correct json format in "logs" section

### DISCUSSION THREAD

**Discussion by hessamheydari40** [mockoon/mockoon#1329] [Ideas]
Hi there!
I have a suggestion to improve the usage of the Mockoon application. I experienced this and that would be a huge benefit for me and others with my usage of mockoon.
![image](https://github.com/mockoon/mockoon/assets/92722681/8e50b422-bb23-40ee-b7e7-f07e357fd2c5)
it this section I want to see the response of API in the correct JSON format to find the desired fields more easily.
thanks.

**Comment by 255kb** [comment]
Hi, i think #368 is a relevant opened issue for this. 

My initial idea in the logs was to show the "raw" data as they enter or leave the server, for better debugging. 
It's true that for proxied requests, it doesn't make real sense. 

Maybe this view and the "view full body in editor" could be always replaced by an editor (better perfs) and JSON content could be beautified, when valid.
Do not hesitate to add your views on the issue too.
